test/fixtures/plugin.colors/mixed.js

```diff
@@ -0,0 +1,45 @@
+module.exports = {
```

Comment:
Wont this test also pass the old one case, see my screenshot from yesterday?

Reply:
@LeeLenaleee fixed